### PR TITLE
Add super user

### PR DESCRIPTION
**Pull request by rnetser**

### CODE REVIEW

**Comment by redhat-qe-bot1** [comment]
Report bugs in [Issues](https://github.com/myakove/github-webhook-server/issues)

The following are automatically added:
 * Add reviewers from OWNER file (in the root of the repository) under reviewers section.
 * Set PR size label.
 * New issue is created for the PR. (Closed when PR is merged/closed)
 * Run [pre-commit](https://pre-commit.ci/) if `.pre-commit-config.yaml` exists in the repo.

Available user actions:
 * To mark PR as WIP comment `/wip` to the PR, To remove it from the PR comment `/wip cancel` to the PR.
 * To block merging of PR comment `/hold`, To un-block merging of PR comment `/hold cancel`.
 * To mark PR as verified comment `/verified` to the PR, to un-verify comment `/verified cancel` to the PR.
        verified label removed on each new commit push.
 * To cherry pick a merged PR comment `/cherry-pick <target branch to cherry-pick to>` in the PR.
    * Multiple target branches can be cherry-picked, separated by spaces. (`/cherry-pick branch1 branch2`)
    * Cherry-pick will be started when PR is merged
 * To build and push container image command `/build-and-push-container` in the PR (tag will be the PR number).
 * To add a label by comment use `/<label name>`, to remove, use `/<label name> cancel`
 * To assign reviewers based on OWNERS file use `/assign-reviewers`
 * To check if PR can be merged use `/check-can-merge`

<details>
<summary>Supported /retest check runs</summary>

 * `/retest build-container`: Retest build-container

</details>

<details>
<summary>Supported labels</summary>

 * hold
 * verified
 * wip
 * lgtm

</details>

File: README.md [hunk 1]
```diff
@@ -10,38 +10,18 @@ Support:
...
-```
+### Local Development
+
+Edit [docker-compose.example.yaml](docker-compose.example.yaml)
+
+Run `docker compose -f .local/docker-compose.yaml up --watch --build`
+
+Open <a href="http://localhost" target="_blank">localhost</a> in browser
+
+The following users configured:
```
Comment (myakove) [inline README.md:21]: missing add docker network if missing.

File: api/api.py [hunk 2]
```diff
@@ -16,25 +15,14 @@
...
-
 @app.route("/api/healthcheck")
 def healthcheck() -> Tuple[Response, int]:
     return jsonify({"status": "ok"}), 200
 
 
 @app.route("/api/@me")
 def get_current_user() -> Tuple[Response, int]:
-    _error = {"error": "Unauthorized", "id": "", "name": ""}
+    _error = {"error": "Unauthorized", "id": "", "admin": False, "name": ""}
```
Comment (myakove) [inline api/api.py:25]: remove `admin` from response.

**Review by myakove — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -10,38 +10,18 @@ Support:
...
-      - </path/to/users.yaml>:/users.yaml
-    environment:
-      - HIVE_CLAIM_FLASK_APP_USERS_FILE=/users.yaml
-      - HIVE_CLAIM_FLASK_APP_SECRET_KEY=<secret_key for flask>
-      - HIVE_CLAIM_FLASK_APP_NAMESPACE=<hive namespace>
-      - HIVE_CLAIM_FLASK_APP_DEBUG=true
-      - AWS_SHARED_CREDENTIALS_FILE=/root/.aws/credentials
-      - KUBECONFIG=/root/.kube/config
-```
+### Local Development
```
Comment (myakove) [inline README.md:13]: Required docker compose version >= 2.22.0

**Review by myakove — CHANGES_REQUESTED** [review]

**Review by myakove — APPROVED** [review]

**Comment by redhat-qe-bot2** [comment]
New container for quay.io/redhat_msi/hive-claim-manager:latest published

**Pull request merged**